### PR TITLE
Adding integer status return to all console commands

### PR DESCRIPTION
**Pull request by msmakouz**

### CODE REVIEW

File: src/Console/Command/Migrate/StatusCommand.php [hunk 1]
```diff
@@ -15,14 +15,14 @@ final class StatusCommand extends AbstractCommand
...
-    public function perform(): void
+    public function perform(): int
     {
         $this->migrator->configure();
 
         if (empty($this->migrator->getMigrations())) {
             $this->writeln('<comment>No migrations were found.</comment>');
 
-            return;
+            return self::FAILURE;
```
Comment (roxblnfk) [inline src/Console/Command/Migrate/StatusCommand.php:25]:
```suggestion
            return self::OK;
```

File: src/Console/Command/Database/ListCommand.php [hunk 2]
```diff
@@ -32,7 +32,7 @@ public function perform(DatabaseConfig $config, DatabaseProviderInterface $dbal)
         if (empty($databases)) {
             $this->writeln('<fg=red>No databases found.</fg=red>');
 
-            return;
+            return self::FAILURE;
```
Comment (roxblnfk) [inline src/Console/Command/Database/ListCommand.php:35]:
```suggestion
            return self::OK;
```

File: src/Console/Command/CycleOrm/MigrateCommand.php [hunk 3]
```diff
@@ -34,13 +33,13 @@ public function perform(
...
         Console $console
-    ): void {
+    ): int {
         $migrator->configure();
 
         foreach ($migrator->getMigrations() as $migration) {
             if ($migration->getState()->getStatus() !== State::STATUS_EXECUTED) {
                 $this->writeln('<fg=red>Outstanding migrations found, run `migrate` first.</fg=red>');
-                return;
+                return self::FAILURE;
```
Comment (roxblnfk) [inline src/Console/Command/CycleOrm/MigrateCommand.php:42]:
```suggestion
                return self::OK;
```

**Review by roxblnfk — COMMENTED** [review]
I think there are no reasons to return the FAILURE exit code in all cases

File: src/Console/Command/Migrate/StatusCommand.php (hunk 1 above)
Comment (msmakouz) [reply src/Console/Command/Migrate/StatusCommand.php]: Changed

File: src/Console/Command/Database/ListCommand.php (hunk 2 above)
Comment (msmakouz) [reply src/Console/Command/Database/ListCommand.php]: Changed

**Pull request merged**